packages/components/src/components/Map.tsx

```diff
@@ -14,6 +14,11 @@ import 'leaflet/dist/leaflet.css';
 import * as L from 'leaflet';
 
 export type MapProps = {
+  tile?: {
```

Comment:
IMO we should support both this flexible approach + the default providers

This `tile` prop (which I think could be renamed to something like `tileLayerSettings` for more clarity) should have an interface like:

```typescript=

type UrlTileLayerSettings = { url: string }

// based on https://github.com/leaflet-extras/leaflet-providers
type MapboxTileLayerSettings = {     
  id: string,
  accessToken: string 
}   

type TileLayerSettings = UrlTileLayerSettings | MapboxTileLayerSettings // ...

...
tileLayerSettings: TileLayerSettings
```

Reply:
added  

```
tileLayerName: TileLayerPreset (MapBox, OpenStreetMap, Esri, custom, .... )
tileLayerOptions: L.TileLayerOptions 
```

cc @demenech